packages/components/Avatar/src/Avatar.types.ts

```diff
@@ -154,6 +157,7 @@ export interface AvatarProps extends IViewProps, AvatarConfigurableProps {
    * using the `getInitials` function.
    *
```

Comment:
I had a query on "`getInitials`()" function. Is it intended to keep it (and other functions in useAvatar) exported and mentioned in documentation?

Reply:
It needs to be exported in order to be tested in a jest file.